csfieldguide/general/context_processors/nav_context.py

```diff
@@ -1,6 +1,6 @@
 """Context processor for displaying chapters in nav bar."""
```

Comment:
I feel like this can be accomplished not as a context processor, therefore not being run on every page.